components/ImageBanner/src/index.scss

```diff
@@ -15,125 +15,126 @@
 }
 
 .denhaag-image-banner__card {
-  align-items: var(--denhaag-image-banner-card-align-items);
-  display: var(--denhaag-image-banner-card-display);
-  height: var(--denhaag-image-banner-card-height);
-  overflow: var(--denhaag-image-banner-card-overflow);
-  position: var(--denhaag-image-banner-card-position);
+  align-items: var(--denhaag-image-banner-card-align-items, end);
+  display: flex;
+  height: var(--denhaag-image-banner-card-height, 378px);
```

Comment:
Ik vraag me puur af, waarom zou je dit component een hoogte meegeven, lijkt me dat dit altijd een aspect-ratio zou moeten zijn 🤔

Reply:
Waarschijnlijk wel, maar for now neem ik over zoals het er staat ;P